airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/metadata/commands.py

```diff
@@ -28,7 +28,7 @@ async def deploy_orchestrator(ctx: click.Context) -> None:
     # Import locally to speed up CLI.
     from pipelines.airbyte_ci.metadata.pipeline import run_metadata_orchestrator_deploy_pipeline
 
-    await run_metadata_orchestrator_deploy_pipeline(
+    return await run_metadata_orchestrator_deploy_pipeline(
```

Comment:
wait lmao is this all we need to make it stop looking like it failed when it succeeded 👀

Reply:
haha yes, thats all it was .